encord/orm/storage.py

```diff
@@ -37,8 +37,9 @@ class StorageLocationName(CamelStrEnum):
     GCP = auto()
     S3 = auto()
     AZURE = auto()
-    OPEN_TELEKOM = auto()
+    OPEN_TELEKOM = auto() # TODO: When enough people have updated their sdk to include S3_COMPATIBLE below
```

Comment:
nit 💅 : TODO doesn't spell out actually required operation

Reply:
Ah supposed to have a "remove" in there. Thx for the catch!